addons/udes_stock_refactoring/models/stock_move.py

```diff
@@ -103,6 +103,9 @@ def _action_refactor(self, stage=None, refactor_action=False):
         refactor_moves = moves.filtered(refactor_lam)
 
         for picking_type, picking_type_moves in refactor_moves.groupby("picking_type_id"):
+            # TECH-DEBT: temporary fix for issue #5172 <https://taiga.unipart.digital/project/udes-x/issue/5172>
+            picking_type_moves = picking_type_moves.filtered(lambda m: m.state != "cancel")
```

Comment:
You can do this outside.

Reply:
As per discussion this might be sorted by changing the order of the `refactor_lam`.